pynest/examples/twoneurons.py

```diff
@@ -22,32 +22,61 @@
 """Two neuron example
 ----------------------------
 
+This script simulates two connected pre- and postsynaptic neurons.
+The presynaptic neuron receives a constant external current.
+And, the membrane potential of both neurons are recorded.
 
 See Also
-~~~~~~~~~~
+~~~~~~~~
 
 :doc:`one_neuron`
 
 """
 
+###############################################################################
+# First, we import all necessary modules for simulation, analysis and plotting.
 
 import nest
 import nest.voltage_trace
 import matplotlib.pyplot as plt
 
+nest.set_verbosity("M_WARNING")
+nest.ResetKernel()
+
+###############################################################################
+# Second, we create the two neurons and the recording device.
+
+neuron_1 = nest.Create("iaf_psc_alpha")
+neuron_2 = nest.Create("iaf_psc_alpha")
+voltmeter = nest.Create("voltmeter")
+
+###############################################################################
+# Third, we set the external current of neuron 1.
+
+neuron_1.I_e = 376.0
+
+###############################################################################
+# Fourth, we connect neuron 1 to neuron 2.
+# Then, we connect a voltmeter to the two neurons.
+# To learn more about the previous steps, please check out the
+# :doc:`one neuron example <one_neuron>`.
+
 weight = 20.0
 delay = 1.0
-stim = 1000.0
 
-neuron1 = nest.Create("iaf_psc_alpha")
-neuron2 = nest.Create("iaf_psc_alpha")
-voltmeter = nest.Create("voltmeter")
+nest.Connect(neuron_1, neuron_2, syn_spec={'weight': weight, 'delay': delay})
```

Comment:
While it's allowed to use both single and double quotes for strings, it should be consistent. We are using double in the single neuron example, so let's go with double here too.
```suggestion
nest.Connect(neuron_1, neuron_2, syn_spec={"weight": weight, "delay": delay})
```